apps/docs/components/components/select.md

```diff
@@ -77,15 +79,15 @@ Since this component uses the native `<select>` element, it inherits its accessi
 
 ## Props
 
-| Prop name         | Type            | Default value | Possible values                        |
-| ----------------  | --------        | ------------- | -------------------------------------- |
-| `value`             | `string`          |            |                                        |
-| `size`              | `SfSelectSize`   | `'base'`          | `'sm'`, `'base'`,`'lg'`                           |
-| `disabled`         | `boolean`         | `false`         |                                        |
-| `invalid`           | `boolean`         | `false`         |                                        |
-| `required`          | `boolean`         | `false`         |                                        |
-| `placeholder`       | `string`          |             |                                        |
-| `wrapperClassName`  | `string`          |             |                                        |
+| Prop name           | Type              | Default value | Possible values                        |
+| ------------------- | ----------------- | ------------- | -------------------------------------- |
+| `modelValue`        | `string`          |               |                                        |
```

Comment:
This needs to be wrapper into Vue/React comments - in react prop name still is `value`, right?

Reply:
good catch thanks

Reply:
fixed https://github.com/vuestorefront/storefront-ui/pull/2933/files#diff-0d5843efe9d67df3645842294a5aec69bac6135a633c42f9a28e9062971c7f21R84